docs/tasks/debug-application-cluster/audit.md

```diff
@@ -70,3 +63,228 @@ the file name (before the file extension) and create a new audit log file.
+When an event is processed it's compared against the list of rules in order.
+The first matching rule sets the audit level of the event. The audit policy is
+defined by the [`audit.k8s.io` API group][audit-api].
+
+An example audit policy file:
+
+```yaml
+rules:
+  # Don't log watch requests by the "system:kube-proxy" on endpoints or services
+  - level: None
+    users: ["system:kube-proxy"]
+    verbs: ["watch"]
+    resources:
+    - group: "" # core API group
+      resources: ["endpoints", "services"]
+
+  # Don't log authenticated requests to certain non-resource URL paths.
+  - level: None
+    userGroups: ["system:authenticated"]
+    nonResourceURLs:
+    - "/api*" # Wildcard matching.
+    - "/version"
+
+  # Log the request body of configmap changes in kube-system.
+  - level: Request
+    resources:
+    - group: "" # core API group
+      resources: ["configmaps"]
+    # This rule only applies to events in the "kube-system" namespace.
+    # The empty string "" can be used to select non-namespaced resources.
+    namespaces: ["kube-system"]
+
+  # Log configmap and secret changes in all other namespaces at the Metadata level.
+  - level: Metadata
+    resources:
+    - group: "" # core API group
+      resources: ["secrets", "configmaps"]
+
+  # Log all other resources in core and extensions at the Request level.
+  - level: Request
+    resources:
+    - group: "" # core API group
+
+  - level: Request
+    resources:
+    - group: "extensions" # Version of group should NOT be included.
+
+  # A catch-all rule to log all other requests at the Metadata level.
+  - level: Metadata
+```
+
+You can use a minimal audit policy file to log all requests at the `Metadata` level:
+
+```yaml
+# Log all requests at the Metadata level.
+rules:
+- level: Metadata
+```
+
+The [audit profile used by GCE][gce-audit-profile] should be used as reference by
+admins constructing their own audit profiles.
+
+### Audit backends
+
+Audit backends implement strategies for emitting events. The [kube-apiserver][kube-apiserver]
+provides a logging and webhook backend.
+
+Each request to the API server can generate multiple events, one when the request is received,
+another when the response is sent, and additional events for log running requests (such as
+watches). The ID of events will be the same if they were generated from the same request.
+
+The event format is defined by the `audit.k8s.io` API group. The `v1alpha1` format of this
+API can be found [here][audit-api] with more details about the exact fields captured.
+
+#### Log backend
+
+The behavior of the `--audit-log-path` flag changes when enabling the `AdvancedAuditing`
+feature flag. This includes some cleanups, such as the `method` reflecting the
+verb evaluated by the [authoriation layer](/docs/admin/authorization/) instead of
+the HTTP verb ("create" instead of "POST"). Also, instead of always generating two
+events per request, events are recorded with an associated "stage." The known stages
```

Comment:
Audit stage isn't specific to the log backend, so I think this should move elsewhere. Maybe under the top level Advanced Auditing